.prettierrc

```diff
@@ -5,5 +5,6 @@
   "useTabs": false,
   "tabWidth": 2,
   "endOfLine": "lf",
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "tailwindConfig": "./tailwind.config.js"
```

Comment:
prettier throws this message:
`Ignored unknown option { tailwindConfig: "./tailwind.config.js" }`